djangosaml2_sp/djangosaml2_sp/settings.py

```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['*',]
```

Comment:
è il caso di creare un settingslocal.py.example da indicare nella doc, così da importare settingslocal.py in settings e fare customizzazioni di configurazione che non introducano parametri nel settings globale